lib/lrama/grammar/parameterizing_rule/resolver.rb

```diff
@@ -25,7 +31,7 @@ def created_lhs(lhs_s_value)
 
         def select_rules(token)
           rules = select_rules_by_name(token.rule_name)
-          rules = rules.select { |rule| rule.required_parameters_count == token.args_count }
+          rules = rules.select { |rule| rule.required_parameters_count == token.args_count && !rule.is_inline }
```

Comment:
[nits] It might be better to extract `rules.select {|rule| !rule.is_inline}` into another method.

Reply:
That's good. I updated this PR: https://github.com/ruby/lrama/pull/369/commits/6928bc856a5358195533ed13bb021f6e6f980ef5